docs/en/Extensions.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: global
+group: Features
+title: Under Storage Extensions
+priority: 7
+---
+
+* Table of Contents
+{:toc}
+
+Alluxio can be extended with the addition of under storage modules at runtime. Under storage extensions can be built as JARs and included at a specific location to be picked up by core Alluxio without the need to restart. This page describes the internals of how extensions in Alluxio work, and provides instructions for building and installing an extension.
```

Comment:
linewidth

Reply:
can you add some statement  from user's perspective, so that readers can easier understand the benefits of supporting UFS extensions